vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -0,0 +1,57 @@
+# How to customize VizroAI
```

Comment:
@nadijagraca I'm not sure if we need a new page for this content. Some of it is already in the FAQ https://vizro.readthedocs.io/projects/vizro-ai/en/latest/pages/explanation/faq/ and I wonder if I should just take what you've added and move it into the relevant Q&A pair (if it's not already stated). WDYT?

Reply:
@stichbury I made a slightly different suggestion below, let me know what you think. That would kill the FAQs alltogther

Reply:
I'll take a look. @nadijagraca I can work on your branch if that's OK and move the content around as needed.